DataFormats/TCDS/src/classes.h

```diff
@@ -0,0 +1,15 @@
+#include "DataFormats/TCDS/interface/TCDSRecord.h"
+#include "DataFormats/Common/interface/Wrapper.h"
+#include "DataFormats/Common/interface/Ref.h"
+#include "DataFormats/Common/interface/RefProd.h"
+
+TCDSRecord tcdsRecord;
```

Comment:
this should be inside the struct below

see https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-20387/22740/llvm-analysis/

Reply:
Indeed.
Not because of the concurrency error reported by the static analyser, but because otherwise we actually instantiate a global TCDSRecord object :-(

Reply:
Thanks for catching this bug. Addressed by commit 43d709d.